kumascript/src/templates.ts

```diff
@@ -108,7 +108,7 @@ export default class Templates {
       });
       return rendered.trim();
     } catch (error) {
-      console.error(`${name} macro failed:`, error);
+      console.error(`The ${name} macro on ${args?.page?.uri} failed:`, error);
```

Comment:
Looks better. I tested in `mdn/content/files/en-us/learn/index.md` (one of the first files we hit on build) by adding a specifications macro and no bcd entry:

```md
---
title: Learn web development
slug: Learn
page-type: landing-page
---

{{Specifications}}
```
Result:

```
The specifications macro on /en-US/docs/Learn failed: Error: /Users/bsmith/Code/forks/yari/kumascript/macros/Specifications.ejs:1
 >> 1| <%
    2| /*
    3| Placeholder to render a specification section with spec_urls from BCD
    4|

No first query argument or 'browser-compat' or 'spec-urls' front-matter value passed
```

Looks good. It reports the slug, not the source path, should we mention this in the console? We could also say:


```md
The specifications macro on URL /en-US/docs/Learn
```

Reply:
Thanks! I'd keep the "URL" word out, but let me see if I can show the file path instead.